src/default.js

```diff
@@ -1,288 +1,81 @@
-			home        : null
-		}
-	},
-	computed : {
-		appsRunning () {
-			return _.filter(this.apps, 'start');
-		},
-		appsPending () {
-			return _.filter(this.apps, ['start', false]);
-		}
-	},
-
-	mounted () {
-		// Start with loading the config
-		this._loadConfig();
-
-		this.$once('afterLoadConfig', () => {
-			this._setupApps();
-		});
-
-		this.$once('afterSetupApps', () => {
-			this._bootApp(_.head(this.apps));
-		});
-
-	},
-
-	methods: {
-
-		/**
-		 * Write apps.json to this.apps
-		 *
-		 * @return Promise
-		 */
-
-		_loadConfig () {
-			$.getJSON('config.json', (config) => {
-				this.config = config;
-				this.apps   = config.apps;
-				this.$emit('afterLoadConfig');
-			}).fail((err) => {
-				if (err.status === 404) {
-					this.$uikit.notification({
-						message: '<strong>config.json missing!</strong><br>Make sure to have this file in your root folder.',
-						status: 'danger',
-						timeout: 0
-					});
-				}
-			});
-		},
-
-
-		/**
-		 * Setup all available apps
-		 */
-
-		_setupApps () {
-
-			_.forEach(this.apps, (app, index) => {
-
-				// TODO: Find better var name for 'foo'
-				requirejs([this.appJS(app.id, 'boot')], ( a ) => {
-
-					a.setup().then((appInfo) => {
-						if (app.id !== appInfo.id) {
-							console.error(`PHOENIX: App ID missmatch! ${app.id} != ${appInfo.id}`);
-							return false;
-						}
-
-						this._registerApp(index, appInfo);
-
-						if (appInfo.start)
-							this.$bus.emit(app.id +':start');
 
-						if (this.apps.length === ++index) {
-							this.$emit('afterSetupApps');
-						}
-					});
+// --- Plugins ----
 
-				}, (err) => {
-					this.warn(err);
-				});
-			});
-		},
+import VueEvents from 'vue-events';
+import VueRouter from 'vue-router';
 
-		/**
-		 * Setup all available apps
-		 *
-		 * @param appInfo (object) return value on setup/boot
-		 */
+Vue.use(VueEvents);
+Vue.use(VueRouter);
 
-		_registerApp ( index, appInfo ) {
+// --- Router ----
 
-			let defaults = {
-				enabled : true,
-				start : false,
-				styles  : []
-			};
+jquery.getJSON('config.json', (config) => {
 
-			let model = _.assignIn(defaults, appInfo);
-			this.apps[index] = model;
-		},
+	let apps = _.map(config.apps, (app) => {
+		return `./apps/${app}/js/${app}.bundle.js`
+	})
 
-		_addStyles ( appId, array ) {
-			let $head = $('head');
-			_.forEach(array, style => {
-				$('<link>', {
-					rel        : 'stylesheet',
-					href       : `apps/${appId}/${style}`,
-					'data-rel' : appId
-				}).appendTo($head);
-			});
-		},
+	requirejs(apps, (a, b, c, d, e) => {
 
-		_removeStyles ( appId ) {
-			$(`link[data-app-id="${appId}"]`).remove();
-		},
+		let plugins  = [];
+		let navItems = [];
+		let routes   = [{
+			path : '/',
+			redirect : '/' + config.apps[0]
+		}];
 
-		/**
-		 * Boot an application
-		 *
-		 * @param obj app with appId as key
-		 * @return Promise
-		 */
+		// @TODO: Need better implementation than [a, b, c, d, e, …]
+		_.forEach([a, b, c, d, e], (app) => {
```

Comment:
can you fix this right away ? at first glance it is very unclear what the letters are about